smart-contracts/package.json

```diff
@@ -32,8 +32,8 @@
     "remix": "remixd -s contracts --remix-ide http://localhost:8080 --read-only",
     "test": "truffle test --network test",
     "lint": "solhint \"contracts/**/*.sol\"",
-    "ci": "npm run lint && npm run test",
-    "build": "truffle compile --reset",
+    "ci": "npm run lint && npm run build && npm run test",
```

Comment:
I'm just not sure if this is the right place to add `npm run build`. I could be mistaken here though.

Reply:
How about a dev command which does `lint` `build` then `test`?  That way we can still use just `test` when rerunning tests again to save time.

That may be better than CI anyways, once we have test coverage that will probably be part of the `ci` command but not something we would run locally.

Reply:
I think that would work, and though we do have a dev command already it's in the Unlock-app directory, so no conflict there.